src/components/BottomButton.jsx

```diff
@@ -4,22 +4,34 @@ import { Button } from 'cozy-ui/react'
 import Bottom from 'components/Bottom'
 import Padded from 'components/Spacing/Padded'
 
+const buttonStyle = {
+  transition: 'transform 0.5s ease'
+}
+
+const styles = {
+  visibleWrapper: {
+    pointerEvents: 'none'
+  },
+  visibleButton: {
+    ...buttonStyle,
+    opacity: 1,
+    transform: `translateY(0px)`
+  },
+  invisibleButton: {
+    ...buttonStyle,
+    opacity: 0,
+    transform: `translateY(100px)`
+  }
+}
```

Comment:
Why didn't you extract these to a css file ?

Reply:
I am tired of writing CSS files for such tiny components... :)